packages/vscode-extension/README.md

```diff
@@ -1,64 +1,88 @@
+6. If you selected the browser-based authentication, press Enter to confirm your NetSuite domain and follow the instructions in the browser prompt. If you selected to save an issued TBA token, do the following:
+    * Enter your account ID.
+    * Enter the token ID of the TBA token you previously issued in NetSuite.
+    * Enter the token secret of the TBA token you previously issued in NetSuite.
+Your account is successfully set up. </br>
+
+To set an account as default for a project, follow these steps:
+1. Open the Command Palette.
+2. Enter **SuiteCloud** and from the dropdown list, select **SuiteCloud: Set Up Account**.
+3. From the dropdown list, select the authentication ID (authID) of the account you want to set as default.
+The selected account is set as default for the project.
+
+<p align="center">
+  <img src="resources/SetUpAccount.gif" alt="animated" />
+</p>
+
+### <font size="5"> <span style="color:gray"> SuiteCloud Commands </font> </span>  
+
+The following SuiteCloud commands are available through the Command Palette:
+
+| Command | Description | Shortcut |
+| --- | --- | --- |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147339580.html" rel="noopener noreferrer" target=blank>Add Dependency References to the Manifest</a>|Adds the defined dependencies to the Manifest file.| Ctrl+Shift+Alt+A |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147328227.html" rel="noopener noreferrer" target=blank>Create SuiteScript Files</a>|Creates a SuiteScript file based on a script type template that also includes any modules you select to extend NetSuite and customize, search for, and process your data. 
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_162938506015.html" rel="noopener noreferrer" target=blank>Create Project</a>|Creates SDF file-based projects to develop ACPs or SuiteApp projects for internal use or for commercial distribution.|
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147342366.html" rel="noopener noreferrer" target=blank>Deploy Project</a>|Deploys the folder containing the project. The project folder is zipped before deployment including only the files and folders referenced in the deploy.xml file. | Ctrl+Alt+D |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_162930381001.html" rel="noopener noreferrer" target=blank>Import Files</a>|Imports files from a NetSuite account to your project. Note that you cannot import files from a SuiteApp.| Ctrl+Alt+F |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_162971830372.html" rel="noopener noreferrer" target=blank>Import Objects</a>|Imports objects from a NetSuite account to your project.| Ctrl+Alt+O |
+|<a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_.html" rel="noopener noreferrer" target=blank>List Files</a>|Lists the files existing in the File Cabinet of your account.|
```

Comment:
Lists the _files existing_ in the File Cabinet of your account.|
or ?
Lists the _existing files_ in the File Cabinet of your account.|

Reply:
Both are correct, but I decided to keep it simple, so I will remove "existing".